atlasdb-client/src/main/java/com/palantir/atlasdb/AtlasDbConstants.java

```diff
@@ -38,6 +38,7 @@
     public static final TableReference DEFAULT_ORACLE_METADATA_TABLE = TableReference.createWithEmptyNamespace("atlasdb_metadata");
 
     public static final String ORACLE_NAME_MAPPING_TABLE = "atlasdb_table_names";
+    public static final String NAME_MAPPING_PK_CONSTRAINT = "pk_" + ORACLE_NAME_MAPPING_TABLE;
```

Comment:
I think we should highlight that this is relating specifically to Oracle i.e. `ORACLE_NAME_MAPPING_PK_CONSTRAINT`. I realise it gets a bit long though :(